client/src/store/modules/list.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
+
+export default {
+  actions: {
+    getLikedList (ctx, videoId) {
+      axios.get(`/api/posts/get-post-likes/${videoId}`)
+        .then((response) => {
+          console.log(response.data.result)
+          ctx.commit('updateLikedList', response.data.result)
+        })
+        .catch(error => {
+          console.log(error)
+        })
+    }
+  },
+
+  mutations: {
+    updateLikedList (state, list) {
+      list.sort((a, b) => {
```

Comment:
https://doesitmutate.xyz/sort/ , please avoid to mutate the original array, make a copy before mutating.